Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='cavestory'
+version='2.6.5-1'
+files=(
+    "https://github.com/nxengine/nxengine-evo/archive/refs/tags/v${version}.zip v${version}.zip a3a1a802eda46f9922065a35624ba94e3823649a7e71012e479ed91fcdb50255"
+    'https://www.cavestory.org/downloads/cavestoryen.zip cavestoryen.zip aa87fa30bee9b4980640c7e104791354e0f1f6411ee0d45a70af70046aa0685f'
+)
+auth_type='sha256'
+depends=(
+    'libjpeg'
+    'libpng'
+    'SDL2'
+    'SDL2_image'
+    'SDL2_mixer'
+    'SDL2_ttf'
+)
+workdir="nxengine-evo-${version}"
+launcher_name='Cave Story'
+launcher_category='Games'
+launcher_command='/usr/local/bin/nxengine-evo'
+icon_file='platform/switch/icon.jpg'
+
+post_fetch () {
+  mkdir -p ${workdir}/build
+}
+
+build () {
+  cd ${workdir}/build
+  cmake -DCMAKE_BUILD_TYPE=Release .. 
+  make
+}
+
+pre_install () {
+  mv CaveStory/Doukutsu.exe ${workdir}/
+  mv CaveStory/data/* ${workdir}/data/
+  rm -rf CaveStory
+  export PATH=$PORT_BUILD_DIR/${workdir}:$PATH
+  cd ${workdir}
+  nxextract  
+}
+
+install () {
+  cd ${workdir}/build 
+  make prefix=/usr/local DESTDIR=$SERENITY_INSTALL_ROOT install
```

Comment:
`run make -C build install`

Those other variables should already be set by `.port_include.sh`

Perhaps prefix is required. but DESTDIR is already set

Reply:
Trying to run it without DESTDIR will attempt to install it to the host's /usr/local/bin.

Reply:
I think you need to pass `"${makeopts[@]}"` as an argument to `make` to make that work. That's the best way to do it since it will always use the configured make options in the future.

Reply:
${installopts[@]} in this case iirc

Reply:
"${makeopts[@]}" by itself will not work. However, setting makeopts to "DESTDIR=$DESTDIR" fixes it.

Reply:
> ${installopts[@]} in this case iirc

${installopts[@]} will also attempt to install to the host's /usr/local/bin

Reply:
@ADKaster @gloof11 You are both right.

https://github.com/SerenityOS/serenity/blob/bfd6deed1e860ce9516c7ca1613150a2c2c75f75/Ports/.port_include.sh#L477

Maybe we should have included `DESTDIR` in `installopts` instead of adding it here as a separate argument? 🤔